test/test_server.py

```diff
@@ -0,0 +1,141 @@
+# -*- coding: utf-8 -*-
+
+import pytest
+from six import add_metaclass
+
+from tango import GreenMode
+from tango import DevState, AttrWriteType
+from tango.server import Device, DeviceMeta
+from tango.server import command, attribute, device_property
+from tango.test_utils import DeviceTestContext
+
+# Pytest fixtures
+from tango.test_utils import state, typed_values, server_green_mode
+state, typed_values, server_green_mode
```

Comment:
I didn't want Flake-8 to complain about unused imports, but there's probably a better way.